back/spec/acceptance/users_spec.rb

```diff
@@ -55,6 +55,48 @@
       end
     end
 
+    get 'web_api/v1/users/check/:email' do
+      let(:email) { 'test@test.com' }
+
+      context 'when a user does not exist' do
+        example_request 'Returns "show_terms"' do
+          expect(status).to eq 200
+          json_response = json_parse response_body
+          expect(json_response[:action]).to eq('show_terms')
+        end
+      end
+
+      context 'when a user exists without a password and has completed registration', document: false do
+        before { create :user_no_password, email: 'test@test.com', registration_completed_at: Time.now }
+
+        example_request 'Returns "continue"' do
+          expect(status).to eq 200
```

Comment:
I'm not a huge fan of `assert_status` but for consistency:
```suggestion
          assert_status 200
```